### ISSUE TITLE
Button to copy and override the final frame keyframe with the first frame keyframe settings

### ISSUE THREAD

**Issue by ivelieu** [thargor6/JWildfire#441]
This would be an easy way to make sure that the starting and ending keyframes have exactly the same property settings, so that the animation loops. As it is right now, replacing the end keyframe with the start keyframe (by deleting the end keyframe then copying the start keyframe to the end) seems to not copy all the keyframe properties.
Now, it's totally possible to go through each flame variation and affine tab and the camera settings to try and figure out which offsets need to be changed so the animation lines up, but this has been tedious for me and it seems simple to implement this action with one button instead. Maybe this is a bug with the keyframe duplicate action, I'm not sure.

In other words, I want to do what this image says with a single button press. 
![image](https://github.com/thargor6/JWildfire/assets/28797826/33fe3803-d4b0-4bb6-9376-aa97011e6edc)
